### ISSUE TITLE
[IOS] Error on build

### ISSUE THREAD

**Issue by bedryck** [defold/extension-admob#40]
Error on build:  Defold: 1.6.3,  extension-admob:  3.4.1
![photo_5343813290985378504_w](https://github.com/defold/extension-admob/assets/22659468/3ccfe520-1baa-47dd-8a0c-0a20a29b0702)

[!] The abstract target Pods is not inherited by a concrete target, so the following dependencies won't make it into any targets in your project:
    - Google-Mobile-Ads-SDK (= 10.14.0)

	at com.defold.extender.ProcessExecutor.execute(ProcessExecutor.java:77)
	at com.defold.extender.services.CocoaPodsService.execCommand(CocoaPodsService.java:1286)
	... 14 more

Log.txt - [log.txt](https://github.com/defold/extension-admob/files/14216312/log.txt)

Without extension - build successfully

**Comment by bedryck** [comment]
Also, I am not sure, but I did a successful build with the current extension version. Last time 21 Jan. 
So I guess problems started after the Defold update

**Comment by britzl** [comment]
The log shows:

```
`GoogleAppMeasurement` requires CocoaPods version `>= 1.12.0`, which is not satisfied by your current version, `1.11.3`
```

We've updated CocoaPods to version 1.12.0.

**Comment by bedryck** [comment]
Sorry I am not good in IOS development. Thank you for explanation.

**Comment by bedryck** [comment]
@britzl Sorry to bother you again, but I can't figure this out. 
I updated CocoaPods to version 1.12.0, and already did a few successful builds. But today when I updated Defold I have such another issue :
[log (3).txt](https://github.com/defold/extension-admob/files/14380612/log.3.txt)
This is all related to the admob, because when I delete it, everything builds successfully.

Also, I a little bit confused what this error means - `Errno::ENOENT - No such file or directory @ rb_sysopen - /Users/ec2-user/.cocoapods/repos/jsdelivr-cocoa/all_pods.txt`

I don't have user on my Mac with name `ec2-user`. 
I tried to reinstall CocoaPods but it doesn't help.

My current versions of cocoapods:

cocoapods (1.12.0)
cocoapods-core (1.12.0)
cocoapods-deintegrate (1.0.5)
cocoapods-downloader (1.6.3)
cocoapods-plugins (1.0.0)
cocoapods-search (1.0.1)
cocoapods-trunk (1.6.0)
cocoapods-try (1.2.0)
